roles/repository/templates/share-config-custom.xml.j2

```diff
@@ -58,7 +58,8 @@
          </action>
       </rule>
    -->
-   <config evaluator="string-compare" condition="CSRFPolicy" replace="{{ csrf_config.enabled }}">
+   <config evaluator="string-compare" condition="CSRFPolicy" replace="true">
```

Comment:
is this documented somewhere? can't find it in the alfresco docs

Reply:
It's documented in the code:
https://github.com/Alfresco/share/blob/master/packaging/distribution/src/main/resources/web-extension-samples/share-config-custom.xml#L39